### PR TITLE
Remove quotes from installation instructions

### LINKED ISSUE #15: readme instructions are incorrect
In the README.MD,

the following command is suggested:

```bash
test -r "~/.dir_colors" && eval $(dircolors ~/.dir_colors)
```

but it won't work ( tested on Debian 10, bash 5.0-4 ).
image

The quotes need to be ommited and the command should be:

```bash
test -r ~/.dir_colors && eval $(dircolors ~/.dir_colors)
```

The problem is explained here: https://serverfault.com/a/417255

### LINKED ISSUE #18: Improper directory colors in iterm/tmux session
There is an incorrect background color for directories with file permissions 0777.  Below is a screenshot.  It actually seems that the theme isn't working at all after going through the installation instructions.

https://imgur.com/gallery/8FRdVKC

I am using iTerm2/tmux/zsh.

This is in my .zshrc: `test -r "~/.dir_colors" && eval $(dircolors ~/.dir_colors)`

Any ideas?

### PR DESCRIPTION
**Pull request by ghost**
This PR includes some minor changes to the README:

- Change heading levels to conform to Markdown standards
- Remove quotes from `"~/.dir_colors"` for symlink creation and testing commands

Fixes #15 
Fixes #18
Related to #11

### CODE REVIEW

**Comment by ghost** [comment]
Fixes #15.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,13 @@
 
 <!--lint disable no-duplicate-headings-->
 
+# 0.2.1
+
+_2020-11-10_
+
+- Removed quotes from "~/.dir_colors" in **README.md** to fix symlink creation issue
+- Change heading levels in **README.md** to conform to Markdown standards (no skipped levels)
+
```
Comment (arcticicestudio) [inline CHANGELOG.md]: It's nice to see you care about change documentations, but the release notes will be generated automatically when preparing a new release version.

```suggestion
```

- [x] Please apply the suggestion or remove the entry again manually.

File: README.md [hunk 2]
```diff
@@ -10,32 +10,32 @@
...
 
 To always use the latest development state of Nord dircolors, [clone the repository][repo] and create a [symbolic link][wiki-symlink] of the `src/dir_colors` file to `~/.dir_colors` in your [home directory][wiki-home_dir] afterwards:
 
 ```sh
-ln -sr "$PWD/src/dir_colors" "~/.dir_colors"
+ln -sr "$PWD/src/dir_colors" ~/.dir_colors
 ```
 
-#### Activation
+### Activation
```
Comment (arcticicestudio) [inline README.md]: Again nice to see you care about correct semantics, but the heading levels have been chosen this way to ensure they fit to GitHubs rendering style. Also using a `h1` tag for “Getting Started“ would make it the title of the repository and causes mismatches when Google scrappers try to create a TOC for the search result entry. Even though there is no other `h1` the SVG banner at the top represents it and I've also started to wrap it into a `h1` tag in other projects to support scrappers/parsers.

- [x] Please revert the heading level changes.

**Review by arcticicestudio — CHANGES_REQUESTED** [review]
Hi @michaelcusick 👋🏼 , thanks for your contribution 👍🏼 

The actual install instruction fix looks good to me, but I've added some review comments.

**Comment by ghost** [comment]
@arcticicestudio Thanks for the review. I just made the changes.

**Review by arcticicestudio — APPROVED** [review]
Thanks again for your contribution 👍🏼

**Review by svengreb — APPROVED** [review]

**Pull request merged**